internal/testutils/setup.go

```diff
@@ -17,3 +18,11 @@ func SetupTest(t *testing.T) *assert.Assertions {
 	ux.NewUserLog(logging.NoLog{}, io.Discard)
 	return assert.New(t)
 }
+
+func SetupTempTestDir() (string, error) {
```

Comment:
Rather than doing this, I suggest using `t.Tempdir()`, which is automatically cleaned up by the `testing` package.

Reply:
Great suggestion, didn't know about this!

Reply:
Implemented.